### DISCUSSION TITLE
failed with exitcode 127 : Command not found : sh: 1: mysqldump: not found

### DISCUSSION THREAD

**Discussion by schel4ok** [spatie/laravel-backup#1174] [Q&A]
I am running php and mariadb in separate docker containers and I have error when try to backup

```
$ php artisan backup:run
Starting backup...
Dumping database laravel...
Backup failed because The dump process failed with exitcode 127 : Command not found : sh: 1: mysqldump: not found
.

Backup failed because: The dump process failed with exitcode 127 : Command not found : sh: 1: mysqldump: not found
```

But there is not error when I try to run `php artisan migrate` or `php artisan db:seed`. So I don't think that php doesn't know where db is located.
I found similar closed issue here https://github.com/spatie/laravel-backup/issues/891, but there is no description how it was resolved

**Comment by Almandeel** [comment]
I have same error 

try this if use ubuntu :

sudo apt install mariadb-client
